src/constants/products.ts

```diff
@@ -240,7 +240,7 @@ export const productCategoryMap: { [productId: number]: ProductCategoryEnum } =
   221: ProductCategoryEnum.YieldOptimizer, // Ether.fi Liquid eBTC Bitcoin LRT Vault
```

Comment:
221 Ether.fi Liquid eBTC Bitcoin LRT Vault - renamed to Ether.fi Liquid eBTC Bitcoin LRT

drops "Vault"